orocos_kdl/tests/jacobiandottest.cpp

```diff
@@ -200,10 +200,12 @@ double compare_Jdot_Diff_vs_Solver(const Chain& chain,const double& dt,const int
     MultiplyJacobian(jdot_by_diff,qdot,jdot_qdot_by_diff);
     
     if(verbose){
-        std::cout << "Jdot diff : \n" << jdot_by_diff<<std::endl;
-        std::cout << "Jdot solver:\n"<<jdot_by_solver<<std::endl;
-        
-        std::cout << "Error : " <<jdot_qdot_by_diff-jdot_qdot_by_solver<<q<<qdot<<std::endl;
+        std::cout << "Jdot diff:" << std::endl << jdot_by_diff << std::endl
+                  << "Jdot solver:" << std::endl << jdot_by_solver << std::endl
+                  << "Error:" << std::endl
+                  << jdot_qdot_by_diff-jdot_qdot_by_solver << std::endl
+                  << q << std::endl
+                  << qdot <<std::endl;
```

Comment:
nit: `std::endl` is not equivalent to `"\n"`, because [`std::endl`](https://en.cppreference.com/w/cpp/io/manip/endl) implies a subsequent [flush](http://www.cplusplus.com/reference/ostream/ostream/flush/) operation and might be less performant than `"\n"` for intermediate line breaks. Both or platform-independent as far as I know.

So if you don't want that other processes' output could be printed between the label and the actual value, it would be preferable to keep `"\n"` to insert a line break, or to even only add a single `std::endl` at the very end. Given that the buffers are large enough, the whole block is then written out at once, instead line by line.

In this particular case the difference is mainly aesthetic.

For reference:
- https://gcc.gnu.org/onlinedocs/libstdc++/manual/streambufs.html#io.streambuf.buffering
- https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rio-endl